src/components/remote_control/remote_sdl_pt.json

```diff
@@ -0,0 +1,2350 @@
+{
```

Comment:
I have seen sdl_preloaded_pt.json and remote_sdl_pt.json, what's the difference between them, and how to make sure any change in one file also get corresponding update in another file?